gradle-jdks-settings/src/test/groovy/com/palantir/gradle/jdks/settings/ToolchainJdksSettingsPluginTest.groovy

```diff
@@ -97,7 +97,11 @@ class ToolchainJdksSettingsPluginTest extends IntegrationSpec {
 
         where:
         // testing for the different gradle versions to make sure the reflection in the settings plugin works
-        gradleVersionNumber << [GradleJdkTestUtils.GRADLE_7_6_VERSION, GradleJdkTestUtils.GRADLE_8_5_VERSION, GradleJdkTestUtils.GRADLE_8_8_VERSION]
+        gradleVersionNumber << [
+                GradleJdkTestUtils.GRADLE_7_6_4_VERSION,
```

Comment:
Change to 7.6.4 here, because com.palantir.platform brings in Jackson 2.18.3 which needs Java 21, which needs Gradle 7.6.4, which [brings support for multi-release jars with Java 21](https://docs.gradle.org/7.6.4/release-notes.html)